docker_signal.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
The wrapper script is simply for convenience.  It does two things: 1) eliminates the need to find the container identifier for the "docker kill" command, and 2) only passes signals which the the worker can handle in the container.  The script is NOT required, but at worst it provides example code for shutdown / restart of the container.